custom_components/tuya_local/devices/dreamegg_nite_1.yaml

```diff
@@ -0,0 +1,305 @@
+name: Baby Sound Machine
```

Comment:
HA naming convention for device and entity names says to only capitalise the first word.